trainer.py

```diff
@@ -93,6 +92,7 @@ def evaluate_model_on_dataloader(
             (avg_loss, accuracy): tuple containing the average loss and
             accuracy across all dataset samples.
         """
+        print('evaluating model...')
```

Comment:
I think we can remove this

Reply:
I added this becuase when training it would evaluate the model without any warning and it looked confusing (suddenly the number of samples drops, and then it happens again (once for the test once for the eval set), i find it adds to the UX

Reply:
well, I guess I don't mind that much, so okay 👍